### PR TITLE
Update Guides to use macOS instead of Mac OS X

### PR DESCRIPTION
**Pull request by ck3g**
### Summary

Update Guides to use macOS instead of Mac OS X

### CODE REVIEW

File: guides/source/development_dependencies_install.md [hunk 1]
```diff
@@ -46,8 +46,7 @@ $ cd rails
 
 The test suite must pass with any submitted code. No matter whether you are writing a new patch, or evaluating someone else's, you need to be able to run the tests.
 
-Install first SQLite3 and its development files for the `sqlite3` gem. Mac OS X
-users are done with:
+Install first SQLite3 and its development files for the `sqlite3` gem. MacOS users are done with:
```
Comment (robin850) [inline guides/source/development_dependencies_install.md]: We should certainly not change the case here. It feels a bit weird to start a sentence with a lower case letter but you can rephrase it like:

> On macOS you are done with

Could you also please wrap this line at 80 chars ? :-)

Comment (ck3g) [inline guides/source/development_dependencies_install.md]: yep. Sure

**Comment by robin850** [comment]
By the way, could you put `[ci skip]` in the message of your commit @ck3g ? :-) This will prevent Travis from running a build, which is not necessary as this is just doc changes, thanks !

**Comment by robin850** [comment]
Thank you ! ❤️

**Pull request merged**